langwatch/src/components/evaluations/wizard/hooks/evaluation-wizard-store/slices/evaluatorNodeSlice.ts

```
@@ -14,6 +14,10 @@ import {
   connectEvaluatorFields,
   createFieldMappingEdges,
 } from "~/components/evaluations/utils/field-mapping";
+import type { Component } from "~/optimization_studio/types/dsl";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Component'. Consider to delete it or rename it.

'Component' is defined here:


(lint/suspicious/noRedeclare)